module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <vector>
+
+constexpr const size_t width = 32;
+constexpr const size_t height = 32;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(std::array<std::array<uint8_t, width>, height>& a);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>>& v);
```

Comment:
Have you considered making aliases for longer types like std::vector<std::pair<uint8_t, uint8_t>>?

Reply:
Please review just module 2 :) I fixed all these issue there.

Reply:
Sorry. Could you change PR to include only things that are to check?

Reply:
I am not sure if it possible to edit an existing PR. The last four commits concerns module 2...